Automattic/jetpack
# Search: re-enable auto-collapsing sidebar in Customberg

In https://github.com/Automattic/jetpack/pull/24399 we removed the auto-collapse of the wp-admin menu in Customberg (aka the Instant Search customization experience).

Unfortunately this caused a menu overlapping issue on WordPress.com sites: https://github.com/Automattic/jetpack/issues/24905#issuecomment-1179760785

Fixes https://github.com/Automattic/jetpack/issues/24905.

#### Changes proposed in this Pull Request:
Re-enable the auto-collapsing sidebar in Customberg to prevent the customization UI being overlapped by the menu.

#### Other information:


#### Jetpack product discussion
Original discussion about the auto-collapsing sidebar in p1HpG7-fTB-p2#comment-54345.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

* Apply this change to your site with a Jetpack Search subscription and with Instant Search enabled.
* Navigate to Customberg (/wp-admin/admin.php?page=jetpack-search-configure).
* Ensure that the wp-admin sidebar automatically collapses in desktop viewports to allow maximum space.

<img width="1567" alt="Screen Shot 2022-07-12 at 14 41 08" src="https://user-images.githubusercontent.com/17325/178398320-abf695f9-1936-412c-9ea8-0da47725b083.png">
